node-bootstrapper/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
fine for now, though might be worth considering a Makefile

Reply:
I agree - using the same build system as the rest of AgentBaker is important. Will change this in a later PR. I have got the testing and linting working with the makefile tho.

Reply:
agree as well.  let align on build process